test_scripts/Smoke/Resumption/ATF_Resumption_3rd_ignition_cycle.lua

```diff
@@ -0,0 +1,161 @@
+    Test:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      Test:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        Test:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:AddCommand()
+  local cid = self.mobileSession:SendRPC("AddCommand", { cmdID = 1, vrCommands = {"OnlyVRCommand"}})
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand", {cmdID = 1, type = "Command",
+                      vrCommands = {"OnlyVRCommand"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("SDL does not perform App resumption when media app tries to resume in 3rd ignition cycle.")
+
+function Test:IGNITION_OFF()
+  IGNITION_OFF()
+end
+
+function Test:Restart_SDL_And_Add_Mobile_Connection()
+  Restart_SDL_And_Add_Mobile_Connection()
+end
+
+function Test:IGNITION_OFF()
+  IGNITION_OFF()
+end
+
+function Test:Restart_SDL_And_Add_Mobile_Connection()
+  Restart_SDL_And_Add_Mobile_Connection()
+end
+
+function Test:Register_And_No_Resume_App()
+  local mobile_session1 = mobile_session.MobileSession(self, self.mobileConnection)
+  local on_rpc_service_started = mobile_session1:StartRPC()
+  on_rpc_service_started:Do(function()
+    config.application1.registerAppInterfaceParams.hashID = self.currentHashID
+    Test:expect_Resumption_Data()
+    commonStepsResumption:RegisterApp(default_app_params)
+    commonStepsResumption:ExpectNoResumeApp(default_app_params):Do(function()
+      StopSDL()
```

Comment:
@istoimenova because we can guarantee that SDL didn't send some notification\request only if SDL was correctly shut downed and didn't sent this notification